ssa/globals.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2025 The GoPlus Authors (goplus.org). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ssa
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/goplus/llvm"
+)
+
+func AddGlobalStrings(pkg Package, globalDatas map[string]string) Package {
```

Comment:
Can it be done with: `pkg.NewVarEx(name, prog.String()).Init(...)` ?

Reply:
pkg.AddGlobalString: use pkg.NewVarEx implement